### PR TITLE
EPMDJ-7717 Added fillProbes method call

### PR DESCRIPTION
**Pull request by ViktoriiaLebedeva**

### CODE REVIEW

File: pt-runner/build.gradle.kts [hunk 1]
```diff
@@ -40,13 +40,13 @@ drill {
 
 repositories {
     mavenLocal()
-    maven("https://dl.bintray.com/drill/drill4j")
+    maven("https://drill4j.jfrog.io/artifactory/dril")
```
Comment (IgorKey) [inline pt-runner/build.gradle.kts:43]: typo in "drill"

File: pt-runner/build.gradle.kts [hunk 2]
```diff
@@ -40,13 +40,13 @@ drill {
...
 }
 
 
 dependencies {
     compileOnly("org.springframework:spring-context:5.1.8.RELEASE")
-    implementation("org.springframework.samples:spring-petclinic:2.1.0") { isTransitive = false }
+    implementation("org.springframework.samples:spring-petclinic:2.1.0.BUILD-SNAPSHOT") { isTransitive = false }
```
Comment (IgorKey) [inline pt-runner/build.gradle.kts:49]: any reasons for this change?

File: src/commonMain/kotlin/com/epam/drill/request/Plugins.kt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package com.epam.drill.request
+
+expect object Plugins {
+    fun fillProbes()
```
Comment (IgorKey) [inline src/commonMain/kotlin/com/epam/drill/request/Plugins.kt:4]: agent doesnt know anything about Probes... what is it? this method should **PROCESS INCOMING NETWORK REQUEST** it shouldn do anything with Probes...

File: pt-runner/build.gradle.kts (hunk 2 above)
Comment (Mazel-Tovr) [reply pt-runner/build.gradle.kts:49]: it's my fault when I published the project to a new repository, I forgot to remove the "build" from the version

**Pull request merged**